# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/FrameClasses.py
# Commit: 6141de941e71
# Full Hash: 6141de941e717cdb5485e1aa1ca00ffcd74f8a81
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-01-30 19:41:15
# Regressor Bug: 1683748
# File Overlap Count: 3
# Description:
#   Bug 1683748 - Support Grid/Flex/Table/Column layout for the rendered legend of a fieldset.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101555
# ==============================================================================

diff -r 3cf752580f1b -r 6141de941e71 layout/generic/FrameClasses.py
--- a/layout/generic/FrameClasses.py	Fri Jan 29 19:20:24 2021 +0000
+++ b/layout/generic/FrameClasses.py	Sat Jan 30 13:47:10 2021 +0000
@@ -45,7 +45,6 @@
     Frame("nsImageFrame", "Image", LEAF),
     Frame("nsInlineFrame", "Inline", NOT_LEAF),
     Frame("nsLeafBoxFrame", "LeafBox", LEAF),
-    Frame("nsLegendFrame", "Legend", NOT_LEAF),
     Frame("nsListControlFrame", "ListControl", NOT_LEAF),
     Frame("nsMathMLFrame", "None", NOT_LEAF),
     Frame("nsMathMLmactionFrame", "None", NOT_LEAF),